PRS5001-6000/PRS5349Hippolyt.xml

```diff
@@ -44,14 +45,28 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
-            <person sex="1">
-               <persName>
-                  
-                  Hippolytus
-               </persName>
-               <occupation> wr. </occupation>
+            <person sex="1" sameAs="wd:Q207113">
+               <persName xml:lang="en" xml:id="n1">Hippolytus of Rome</persName>
+               <persName xml:lang="gez" xml:id="n2">ሄጶሊጦስ፡</persName>
+               <persName xml:lang="gez" corresp="#n1">Heṗoliṭos</persName>
```

Comment:
why n2 when this is the same as n1? Also n4 is the same, they should all be linked as "corresp"
the transcription should have the type "normalized"

normally we would have
```
 <persName xml:lang="gez" xml:id="n1">ሄጶሊጦስ፡</persName>
               <persName xml:lang="gez" corresp="#n1" type="normalized">Heṗoliṭos</persName>
               <persName xml:lang="en" corresp="#n1" type="main">Hippolytus of Rome</persName>
```

You need type "main" to have the English form the first one printed in the app (not Heṗoliṭos is now)

Reply:
@eu-genia is right, of course, see the Cyprian example in the guidelines: https://betamasaheft.eu/Guidelines/?id=persons
But actually (also following the Cyprian example), I would encode the English name as 
  `<persName xml:lang="en" xml:id="n2" type="main">Hippolytus of Rome</persName>`, because "Hippolytus of Rome" is not a translation of ሄጶሊጦስ፡.